specs/disk_inventory_etcd_structure.adoc

```diff
@@ -0,0 +1,138 @@
+= Disk inventory etcd structure
+
+This specification specifies the way in which disk inventory has to be structured
+in tendrl's central store and the details that has to be stored in the disk
+inventory
+
+== Problem description
+
+We should have a proper structure to our disk inventory in such a way that we
+store all the needed information in a easily consumable way.
+
+== Use Cases
+
+* This spec provides guidelines about how the disk information is to be
+stored in etcd.
+
+== Proposed change
+
+The disk inventory details will basically stored under 2 sections as follows:
+
+1. /nodes/<node-id>/disks/
+
+* In this location we store all the physical disks available on the node
+with a unique key which is a combination of manufacturer-id+vendor-id+
+device-serial-number this would be unique across tendrl. And this information
+should be available from hwinfo tool.
```

Comment:
@r0h4n @brainfunked VM disk does not have serial-number, we can use this key combination 

and hwifo not giving manufacturing_id also,   
output of hwifo from VM disk:
  Unique ID: WiMi.Fxp0d3BezAE
  Parent ID: vkLc.+FFPFBVXZu6
  SysFS ID: /class/block/vdh
  SysFS BusID: virtio9
  SysFS Device Link: /devices/pci0000:00/0000:00:0e.0/virtio9
  Hardware Class: disk
  Model: "Disk"
  Driver: "virtio-pci", "virtio_blk"
  Driver Modules: "virtio_pci", "virtio_blk"
  Device File: /dev/vdh
  Device Files: /dev/vdh, /dev/disk/by-id/virtio-78065dbf-92db-4574-8, /dev/disk/by-path/virtio-pci-0000:00:0e.0
  Device Number: block 252:112-252:127
  Geometry (Logical): CHS 31207/16/63
  Size: 31457280 sectors a 512 bytes
  Capacity: 15 GB (16106127360 bytes)
  Config Status: cfg=new, avail=yes, need=no, active=unknown
  Attached to: #27 (Storage controller)